src/main/java/org/entando/entando/keycloak/filter/KeycloakFilter.java

```diff
@@ -62,10 +68,21 @@ private void doLogin(final HttpServletRequest request, final HttpServletResponse
         final String stateParameter = request.getParameter("state");
         final String redirectUri = request.getRequestURL().toString();
         final String redirectTo = request.getParameter("redirectTo");
+        final String error = request.getParameter("error");
+        final String errorDescription = request.getParameter("error_description");
+
+        if (StringUtils.isNotEmpty(error)) {
+            if ("unsupported_response_type".equals(error)) {
+                log.error(errorDescription + ". For more details, refer to the wiki " + wiki(KeycloakWiki.EN_APP_STANDARD_FLOW_DISABLED));
```

Comment:
The error_description query param provided by Keycloak has enough details already, didn't make sense to add anything else.